tests/performance_tests/README.md

```diff
@@ -902,6 +902,7 @@ The Performance Test Basic generates, by default, a report that details results
     "ForksCount": 0
   },
   "args": {
+    "rawCmdLine ": "/home/leap/build/tests/performance_tests/performance_test_basic.py -v -p 1 -n 1 --target-tps 20 --tps-limit-per-generator 10 --test-duration-sec 5 --clean-run",
```

Comment:
This report in the README is generated with the following configuration, could you re-run and update with that configuration please.

`.build/tests/performance_tests/performance_test.py --test-iteration-duration-sec 10 --final-iterations-duration-sec 30 --calc-producer-threads lmax --calc-chain-threads lmax --calc-net-threads lmax`